src/components/join-flow/join-flow-step.jsx

```diff
@@ -12,19 +16,22 @@ const JoinFlowStep = ({
 }) => (
     <form onSubmit={onSubmit}>
         <div>
-            {title && (
-                <h2>
-                    {title}
-                </h2>
-            )}
-            {description && (
-                <p>
-                    <span>
-                        {description}
-                    </span>
-                </p>
-            )}
-            {children}
+            <ModalInnerContent className="join-flow-inner-content">
+                {title && (
+                    <ModalTitle
+                        className="join-flow-title"
+                        title={title}
+                    />
+                )}
+                {description && (
+                    <p>
+                        <span>
```

Comment:
do we need both the p and the span? Seems like the span would be unnecessary here.

Reply:
Will fix this in a follow-up.